### PR TITLE
WIP: Test serverless-operator changes for SKIP_OPERATOR_SUBSCRIPTION option

### PR DESCRIPTION
**Pull request by chetna14manku**

### CODE REVIEW

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip-operator-subs-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip-operator-subs-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip-operator-subs-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip-operator-subs-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip_option-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip_option-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip_option-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip_option-ocp4.16-rc-serverless-p2p-rosa

File: ci-operator/config/chetna14manku/serverless-operator/chetna14manku-serverless-operator-skip_option__ocp4.16-rc.yaml [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.16"
+      REGION: us-west-2
+      SLEEP_DURATION: 30m
+    test:
+    - ref: wait
+    - as: mesh-e2e
+      cli: latest
+      commands: SKIP_OPERATOR_SUBSCRIPTION=true ROSA=true make mesh-e2e
```
Comment (mgencur) [inline ci-operator/config/chetna14manku/serverless-operator/chetna14manku-serverless-operator-skip_option__ocp4.16-rc.yaml:166]: I left a comment here: https://github.com/chetna14manku/serverless-operator/commit/750dbec59841177b6db16aaf0f696a0871439f64#r143508832 
We could maybe skip passing the ROSA param

**Review by mgencur — COMMENTED** [review]
This looks pretty good! 
We're just hitting https://issues.redhat.com/browse/OCPBUGS-35731 and all the tests fail because of that. We haven't figured a workaround yet but I plan to work on it this week.

File: ci-operator/config/chetna14manku/serverless-operator/chetna14manku-serverless-operator-skip_option__ocp4.16-rc.yaml (hunk 1 above)
Comment (chetna14manku) [reply ci-operator/config/chetna14manku/serverless-operator/chetna14manku-serverless-operator-skip_option__ocp4.16-rc.yaml:166]: Thanks for review, I'll update the serverless-operator code to use the `oc` command to get `rosa` cluster type

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip_option-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
/pj-rehearse periodic-ci-chetna14manku-serverless-operator-skip_option-ocp4.16-rc-serverless-p2p-rosa

**Comment by chetna14manku** [comment]
Closing PR, done testing changes.

**Pull request closed without merging**